Tasks/VsTest/runvstest.ts

```diff
@@ -48,6 +52,17 @@ if (osPlat !== 'win32') {
     }
 }
 
+function isMultiConfigOnDemandRun(): boolean {
```

Comment:
move to helper class

Reply:
Why is that needed?

Reply:
All such functions we have in that helper.ts file

Reply:
This file is clean enough for now.